app/components/organisms/Login.tsx

```diff
@@ -0,0 +1,43 @@
+import {
+  Box,
+  Heading,
+  HStack,
+  Stack,
+  VisuallyHidden,
+  Image,
+} from '@chakra-ui/react'
+import LoginForm from '../molecules/LoginForm'
+
+const Login = () => {
+  return (
+    <Stack
+      mx={{ base: '4', sm: '16', md: 'unset' }}
+      my="auto"
+      borderRadius={{ base: 'xl', md: 'unset' }}
+      w={{ md: 'xl' }}
+      height={{ base: '1/2', md: 'full' }}
+      p={{ base: 8, md: 16 }}
+      bgColor="white"
+      position="relative"
+      flexDirection="column"
+      gap={4}
+      justifyContent="center"
+    >
+      <HStack justifyContent="center" gap={2}>
+        <Image src="/images/chest.png" />
```

Comment:
```suggestion
        <Image src="/images/chest.png" aria-hidden="true" />
```

et pourquoi pas un width et height en attribut aussi pour éviter un éventuel layout shift

Reply:
Faut voir chakra comme un UIKit